app/src/main/java/com/endava/myendava/activities/SignInActivity.java

```diff
@@ -3,26 +3,36 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.widget.Button;
+import android.widget.CheckBox;
 import android.widget.EditText;
+import android.widget.ProgressBar;
+import android.widget.Toast;
 
 import androidx.annotation.Nullable;
-import androidx.appcompat.app.AppCompatActivity;
 
 import com.endava.myendava.R;
 import com.endava.myendava.app.ApplicationServiceLocator;
 import com.endava.myendava.utils.MySharedPreferences;
+import com.endava.myendava.viewmodels.SignInViewModel;
 
 import javax.inject.Inject;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.Unbinder;
 
-public class SignInActivity extends AppCompatActivity {
+public class SignInActivity extends BaseActivity {
+
+    public static final String INVALID_EMAIL_MESSAGE = "Invalid email";
+
```

Comment:
usually there is no space between attributes of same type (applies to @Inject and @BindView also)